Assets/Scripts/Models/InventoryManager.cs

```diff
@@ -6,100 +6,89 @@
 // file LICENSE, which is part of this source code package, for details.
 // ====================================================
 #endregion
-
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using MoonSharp.Interpreter;
-using UnityEngine;
 
 [MoonSharpUserData]
 public class InventoryManager
 {
-    // This is a list of all "live" inventories.
-    // Later on this will likely be organized by rooms instead
-    // of a single master list. (Or in addition to.)
-    public Dictionary<string, List<Inventory>> inventories;
+    private static readonly string InventoryManagerLogChanel = "InventoryManager";
 
     public InventoryManager()
     {
-        inventories = new Dictionary<string, List<Inventory>>();
+        Inventories = new Dictionary<string, List<Inventory>>();
     }
 
-    public bool PlaceInventory(Tile tile, Inventory inv)
+    public Dictionary<string, List<Inventory>> Inventories { get; private set; }
```

Comment:
I think that Inventories should be completely private. I can't switch the dictionary for another one, but I can still grab the instance and add/remove items without using the proper functions. For anything that is using the dictionary directly, we can add a method.